sql_metadb/derived_tables/runlist.txt

```diff
@@ -26,6 +26,7 @@ instance_formats.sql
 instance_editions.sql
 instance_languages.sql
 instance_nature_content.sql
+instance_tags.sql
```

Comment:
There are no dependencies on other derived tables. So it should be sorted alphabetically. See also discussion in PR #638